src/System.Data.SqlClient/tests/ManualTests/SQL/AsyncTest/AsyncTest.cs

```diff
@@ -91,5 +92,99 @@ public static void FailureTest()
             readerTask.Result.Dispose();
             con.Close();
         }
+
+
+        [CheckConnStrSetupFact]
+        public static void TestReadAsync()
+        {
+            CompareReadSyncAndReadAsync();
+        }
+
+        private static async void CompareReadSyncAndReadAsync()
```

Comment:
This method doesn't need to be async

Reply:
It contains async operation line, and not able to remove `async` keyword in the method signature.
```
double elapsedAsync = await RunReadAsync(sql);
```

Reply:
@saurabh500 Never mind. I see your intention.